app/editor/src/features/content/form/interfaces/IContentForm.tsx

```diff
@@ -22,7 +22,7 @@ export interface IContentForm {
   ownerId: number | '';
   contentTypeId: number;
   mediaTypeId: number;
-  licenseId: number;
+  licenseId?: number;
```

Comment:
The DB requires a license, so it's odd to allow it to be nullable on the form.  Is this to force the user to select a value?